packages/dev/core/src/Particles/baseParticleSystem.ts

```diff
@@ -746,52 +742,37 @@ export class BaseParticleSystem implements IClipPlanesHolder {
      * Creates a Point Emitter for the particle system (emits directly from the emitter position)
      * @param direction1 Particles are emitted between the direction1 and direction2 from within the box
      * @param direction2 Particles are emitted between the direction1 and direction2 from within the box
-     * @returns the emitter
      */
     public createPointEmitter(direction1: Vector3, direction2: Vector3): PointParticleEmitter {
-        const particleEmitter = new PointParticleEmitter();
-        particleEmitter.direction1 = direction1;
-        particleEmitter.direction2 = direction2;
-
-        this.particleEmitterType = particleEmitter;
-        return particleEmitter;
+        throw new Error("Method not implemented.");

```

Comment:
All the throws should be `throw _WarnImport("...");` I believe

Reply:
it is not a warm import as you cannot call that method at all for ThinParticleSystem (but it is still needed for hierarchy)